terraform-provider-intro/order-update.md

```diff
@@ -0,0 +1,79 @@
+Change your order by updating the order resource through Terraform. In your `main.tf`, update the coffee quantity in `hashicups_order.edu` block.
+
+Change the first coffee item from `2` to `3` and change the second coffee item from `2` to `1`.
+
+```
+resource "hashicups_order" "edu" {
+  items {
+    coffee {
+      id = 3
+    }
+    quantity = 3
+  }
+  items {
+    coffee {
+      id = 2
+    }
+    quantity = 1
+  }
+}
+```
+
+Run `terraform apply` to update the order. Notice how the execution plan reflects the order change. Terraform is able to determine that these changes could be done in place rather than destroying the current order and creating a new one. This is because the modified property ([quantity](https://github.com/hashicorp/terraform-provider-hashicups/blob/master/hashicups/resource_order.go#L63)) does not have `ForceNew` set to `true`.
```

Comment:
Is "forcenew" an option in every provider? Would be cool to see the differences in cloud providers that utilize this, at least a link.

Reply:
It's an option that you can define in the resource! I'll link an [AWS ec2 code](https://github.com/terraform-providers/terraform-provider-aws/blob/a15140e1184d201c9cf4cfe46f154a48c4a08958/aws/resource_aws_instance.go#L45-L49) as an example, if you change the AMI, it destroys the existing one and creates a new one.